docs/topics/feed-exports.rst

```diff
@@ -233,6 +234,13 @@ The serialization format to be used for the feed. See
 
 .. setting:: FEED_EXPORT_FIELDS
 
+FEED_EXPORT_ENCODING
+--------------------
+
+The encoding to be used for the feed. Defaults to UTF-8.
```

Comment:
Not sure. The default value is `None`, which means is the backward-compatible default behavior (current master, before the merge), which is:
- `\uXXXX` escape sequences for JSON
- UTF-8 for XML and CSV
(if I'm not mistaken)

And users can change to UTF-8 for JSON instead of `\uXXXX` with `FEED_EXPORT_ENCODING = 'utf-8'`

If set, and different from `"utf-8"`, it changes behavior for all exporters.

I don't have good phrasing right now. Maybe @eliasdorneles has ideas?